static/js/context-menu.js

```diff
@@ -1220,6 +1227,10 @@ var Hover = new (class Hover {
     this.hoveredElem = elem;
   }
 
+  onClick(elem) {
+    this.updateHoverState(elem);
```

Comment:
As mentioned above, please rename this to `onSymbolClicked`, and move the comment for the reasoning here.
Also please add a comment that this is called by ContextMenu, to clarify why this is not directly an event listener.